cdap-ui/app/cdap/components/Experiments/FieldLevelLineage/Table/Table.css

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+.table {
+  width: 170px;
+  border: 1px solid #bbbbbb;
+  margin: 10px;
+}
+
+.row {
+  border-bottom: 0.5px solid lightgray;
+  height: 20px;
+  padding-left: 10px;
+  margin-right: 0;
+  margin-left: 0;
+}
+
+.table-header {
+  border-bottom: 2px solid #bbbbbb;
+  height: 40px;
+  padding-left: 10px;
```

Comment:
Just a note: When we convert this to css-in-js the spacing should be based on units from `material-ui` and colors should be variables from `ThemeWrapper`